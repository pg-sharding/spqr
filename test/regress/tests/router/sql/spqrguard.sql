
SET __spqr__maintain_params TO TRUE;

SET allow_system_table_mods TO true;

CREATE EXTENSION spqrguard;
DROP EXTENSION spqrguard;
