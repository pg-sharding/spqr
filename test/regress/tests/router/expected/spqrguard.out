SET __spqr__maintain_params TO TRUE;
SET allow_system_table_mods TO true;
CREATE EXTENSION spqrguard;
NOTICE: send query to shard(s) : sh1,sh2,sh3,sh4
DROP EXTENSION spqrguard;
NOTICE: send query to shard(s) : sh1,sh2,sh3,sh4
